tests/other/test_load_split_module.post.js

```diff
@@ -1,6 +1,6 @@
 Module["loadSplitModule"] = function(deferred, imports, prop) {
     console.log('Custom handler for loading split module.');    
-    if (prop) {
+    if (prop && !isNaN(parseInt(prop))) {
```

Comment:
Why is this parsing necessary? Are there ever non-number placeholder names?

Reply:
hmm just wanted to check that we are passing in the correct variable here. let me know if i should remove it.

Reply:
Yes, I think it would be sufficient and simpler to just unconditionally log here.